examples/analytics-vendors.amp.html

```diff
@@ -27,6 +27,43 @@
 <div id="container">
 Container for analytics tags. Positioned far away from top to make sure that doesn't matter.
 
+<!-- Acquia Lift example -->
+<amp-analytics type="acquialift" id="acquialift">
+<script type="application/json">
+{
+  "requests":
+    "customEvent": "${basicCapture}&en=${eventName}"
+  },
+  "vars": {
+    "siteId":"my-acquia-lift-site-id",
+    "accountId": "MYACCOUNTID",
+    "decisionApiUrl": "us-east-1-decisionapi.lift.acquia.com"
+  },
+  "triggers": {
+    "pageLoad": {
+      "on": "visible",
+      "request": "pageview"
+    },
+    // Sends a Click-Through event.
```

Comment:
Sorry for nitpick, These comments will make the json parsing to fail. To make sure everythign is working, have you also done 
`gulp` and then gone to /examples/analytics-vendors.amp.html?